plenum/__init__.py

```diff
@@ -68,7 +68,5 @@ def setup_plugins():
 setup_plugins()
 
 
-from __metadata__ import *  # noqa
-
 from plenum.common.jsonpickle_util import setUpJsonpickle   # noqa: E402
```

Comment:
Changes in #774 made package uninstallable because __metadata__ is not included into the package. This place is buggy as well but it doesn't solve the problem, moreover `plenum.__version__` and other metadata won't be available if we remove that import.